src/logic/otp2/controller.ts

```diff
@@ -515,6 +438,42 @@ async function searchFlexible(
     }
 }
 
+async function searchFlexibleEgress(
+    searchParams: GetTripPatternsQueryVariables,
+    extraHeaders: Record<string, string>,
+): Promise<{
+    tripPatterns: TripPatternParsed[]
+    queries: GraphqlQuery[]
+}> {
+    const getTripPatternsParams = {
+        ...searchParams,
+        modes: {
+            ...searchParams.modes,
+            egressMode: StreetMode.Flexible,
+        },
+    }
+
+    const comment = 'Search ordinary with flexible Egress'
```

Comment:
Trenger søket en kommentar? 😮

Reply:
Vi har lagt til kommentar flere steder på søkene for å se hvilke søk som kjøres i loggen